hardhat.config.ts

```diff
@@ -98,29 +98,29 @@ const config : HardhatUserConfig = {
     enabled: false,
   },
   networks: {
-    mainnet: {
-      url: `${process.env.MAINNET_RPC_URL}`,
-      gasPrice: 80000000000,
-    },
-    sepolia: {
-      url: `${process.env.SEPOLIA_RPC_URL}`,
-      timeout: 10000000,
-      // accounts: [ // Comment out for CI, uncomment this when using Sepolia
-      //   `${process.env.TESTNET_PRIVATE_KEY_A}`,
-      //   `${process.env.TESTNET_PRIVATE_KEY_B}`,
-      //   `${process.env.TESTNET_PRIVATE_KEY_C}`,
-      //   `${process.env.TESTNET_PRIVATE_KEY_D}`,
-      //   `${process.env.TESTNET_PRIVATE_KEY_E}`,
-      //   `${process.env.TESTNET_PRIVATE_KEY_F}`,
-      // ],
-      // // Must have to avoid instead failing as `invalid length for result data` error
-      // throwOnCallFailures: false, // not sure if this even works
-    },
-    devnet: {
-      // Add current URL that you spawned if not using automated spawning
-      url: `${process.env.DEVNET_RPC_URL}`,
-      chainId: 1,
-    },
+    // mainnet: {

```

Comment:
Should this be uncommented? Why did you comment out all the networks?

Reply:
Someone was on the way of deploy. Got an advise to comment everything until we deploy prod

Reply:
Okay. You don't need to change it, but unless you provide `--network [value]` as a parameter to the tests in the CLI (or to any hardhat command) it won't use these details.

Reply:
just comment out PRIVATE_KEY type vars in there so you don't get errors.